Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get dist-upgrade -y
   software-properties-common \
   && rm -rf /var/lib/apt/lists/*
 
-RUN if [ "${PYTHON_VERSION}" != "3.6" ] ; \
```

Comment:
A couple of dumb questions about other lines in this file that I don't fully understand or remember what they do:

1. In line 42, do we still need the `= 3.8` condition?
2. In line 45, does that `2.7` refer to the Python version, and does it still make sense? Should it change?

Reply:
Line 42 still applies, debian distros for Python 3.8 and newer split `python` package and into `python` and `python-distutils`. We need `python-distutils` to build some packages from source.

In line 45 we indeed don't need 2.7 anymore, it meant last version of pip that works on 2.7 and we don't care about 2.7 anymore. I changed that.